src/Time/Tags.hpp

```diff
@@ -63,6 +63,19 @@ struct HistoryEvolvedVariables : db::DataBoxPrefix {
       std::deque<std::tuple<::Time, db::item_type<Tag>, db::item_type<DtTag>>>;
 };
 
+/// \ingroup DataBoxTags
+/// \ingroup TimeGroup
+/// Prefix for TimeStepper boundary history
+///
+/// \tparam Key type identifying a boundary
+/// \tparam Tag tag for boundary variables
+template <typename Key, typename Tag>
```

Comment:
Should the order of the template parameters be reversed for the `add/remove_prefix` metafunctions?

Reply:
These aren't usable with the prefix modifiers because that would try to create a Variables containing unordered_maps.